cloudapi/logs.go

```diff
@@ -111,18 +113,26 @@ func (c *Config) getRequest(referenceID string, start time.Duration) (*url.URL,
 func (c *Config) StreamLogsToLogger(
 	ctx context.Context, logger logrus.FieldLogger, referenceID string, start time.Duration,
 ) error {
-	u, err := c.getRequest(referenceID, start)
-	if err != nil {
-		return err
-	}
+	var conn *websocket.Conn
 
-	headers := make(http.Header)
-	headers.Add("Sec-WebSocket-Protocol", "token="+c.Token.String)
+	dial := func() error {
+		u, err := c.getRequest(referenceID, start)
```

Comment:
`getRequest` is using `start` as a relative point from `Now`, in this way, when the dial function is invoked all the previous logs should be skipped. We are passing `0` from the cmd so, currently, the final query it's matching `Now`. 

https://github.com/grafana/k6/blob/master/cloudapi/logs.go#L101-L104

Reply:
You are correct. Sorry, I was confused by the name of the parameter (which is also my fault given that I named it ;) ).

Given that though it means that we won't even try to get the messages between when we disconnect and when we finally reconnect. 
So we either:
1. Rename the parameter to be better named and less confusing and open an issue to fix the above later
2. Fix this now so that `start` is the actual parameter `start` to the loki [endpoint](https://grafana.com/docs/loki/latest/api/#get-lokiapiv1tail)  which is the time from which loki should start returning logs